src/providers/DocumentSymbolProvider.ts

```diff
@@ -20,7 +20,7 @@ export class DocumentSymbolProvider implements vscode.DocumentSymbolProvider {
 				const line = document.lineAt(i);
 				let tokens: string[] = line.text.trim().split('(');
 				let lineText = line.text.trim().toLowerCase();
-				let symbol = tokens[0].trim() || "undef";
+				let symbol = tokens[0].trim() || "undef"; // Kludge "cannot be falsy"
```

Comment:
This kludge is to just move on with life, and provide best effort. Ideally we would not have to do this, but, w/e for now.